DemoApp/Sources/Views/CallView/DemoCallView.swift

```diff
@@ -38,7 +38,7 @@ struct DemoCallView<ViewFactory: DemoAppViewFactory>: View {
         viewFactory
             .makeInnerCallView(viewModel: viewModel)
             .onReceive(viewModel.callSettingsPublisher) { _ in
-                updateMicrophoneChecker()
+                Task { await updateMicrophoneChecker() }
```

Comment:
Not important for now but maybe at some point we should add a async supported `onReceive`.

Reply:
That's a nice idea